content/guides/containers/managing-container-images-registry.md

```diff
@@ -0,0 +1,67 @@
+
+* Scans images for CVEs in conjunction with a number of scan providers including CentOS/Clair, Aqua/Trivy, Anchore/Engine, and DoSec to detect container images with vulnerabilities. 
+
+* Integrates with Active Directory/LDAP and OIDC to provide role-based access control so that you can securely store images behind your firewall.
+
+* Signs images as trusted by using [Notary](https://github.com/theupdateframework/notary). You can also set policies that prevent vulnerable or untrusted images from being used in production.
+
+### Vulnerability Scanning
+
+Containers often use base images of operating systems like Ubuntu and CentOS from a public image repository, such as DockerHub. The packages of an operating system and the applications on it, however, can contain vulnerabilities.
+
+Vulnerability scanning helps detect known vulnerabilities to reduce the risk of security breaches. Identifying a vulnerability on an image and keeping the vulnerability from going into production reduces the attack surface of a containerized application.
+
+The Harbor image registry scans images with Clair or Trivy (default options) to help prevent images with known vulnerabilities from running in production. With Harbor, you can automate the scanning of images or scan them manually.
+
+Scanners work by continuously importing metadata about vulnerabilities from various sources, including CVEs, and storing the metadata in a database. When Harbor scans an image, Harbor checks the database through the appropriate API and produces a list of vulnerabilities.
+
+### Establishing Content Trust with Notary
+
+Because images are the building blocks of a containerized application, a security best practice is to use signed container images from a trusted registry. Harbor establishes trust by letting publishers sign images when they push them into the repository and by preventing unsigned images from being pulled from the repository.
+
+The Notary services of the Harbor private registry can furnish project-level content trust to container images to ensure that only trusted images are used as developers create their own layered images, automate the building of a container, or select images for use as the building blocks of an application.
+
+### Image and Registry Security in the Software Pipeline
+
+Harbor works in the context of a development pipeline to manage and secure container images. The following diagram illustrates Harbor's role in a development pipeline:
+
+![The Harbor container registry manages and secures container images.](/images/guides/containers/harbor-registry-security.png)
```

Comment:
Might be worth considering putting this in a "diagrams" subdirectory to add padding since it's pretty close to the edge of the image. Not a big deal if not though.